docs/docs/tutorials/get-started/further-readings.md

```diff
@@ -3,3 +3,32 @@ sidebar_position: 6
 ---
 
 # Further readings
+
+### [⚔️ Protect your API](../../recipes/protect-your-api/README.md)
```

Comment:
should be heading 2 below heading 1?

Reply:
it's intended since heading 2 is too large, i talked to rany to design some cards for this